utils/audit/files/audit.init

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh /etc/rc.common
+# Copyright (c) 2014 OpenWrt.org
+
+START=11
+
+USE_PROCD=1
+PROG=/usr/sbin/auditd
+
+start_service() {
+	mkdir -p /var/log/audit
+	procd_open_instance
+	procd_set_param command "$PROG" -n
+	procd_set_param respawn
+	procd_close_instance
+	test -f /etc/audit/rules.d/audit.rules && /usr/sbin/auditctl -R /etc/audit/rules.d/audit.rules
```

Comment:
Be aware that you cannot know at this point if `auditd` is actually running yet. Does `auditctl` wait in case procd is still busy with other things or auditd still busy starting up?

Reply:
I doubt auditctl will wait. Does procd provide a syntax to wait for a daemon to start? Should I break this into two init scripts?

Reply:
auditctl does not strictly depend on auditd AFAIK. Instead it depends on audit support in the Kernel. In theory one could have auditctl and not auditd. The audit messages would end up in Dmesg instead of /var/log/audit. Maybe package auditctl separately so that one can opt-out of auditd?

Reply:
... dmesg instead of /var/log/audit, or otherwise whatever is reading the events from the netlink audit socket.

Reply:
I split the package into audit and audit-utils.